<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_03">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton::hover
{
color: rgb(0, 0, 0);
background-color : lightgreen;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}
QPushButton
{
color: rgb(0, 255, 255);
	background-color: rgb(0, 0, 255);
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}</string>
       </property>
       <property name="text">
        <string>Button_03</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_04">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton::hover
{
color: rgb(0, 0, 0);
background-color : lightgreen;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}
QPushButton
{
color: rgb(0, 255, 255);
background-color: rgb(255, 0, 255);
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}</string>
       </property>
       <property name="text">
        <string>Button_04</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_01">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton::hover
{
color: rgb(0, 0, 0);
background-color : lightgreen;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}
QPushButton
{
color: rgb(0, 255, 255);
background-color: rgb(170, 0, 0);
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}</string>
       </property>
       <property name="text">
        <string>Button_01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_02">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton::hover
{
color: rgb(0, 0, 0);
background-color : lightgreen;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}
QPushButton
{
color: rgb(0, 255, 255);
	background-color: rgb(85, 85, 0);
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}</string>
       </property>
       <property name="text">
        <string>Button_02</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>궁서</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::hover
{
color: rgb(0, 0, 0);
background-color : lightgreen;
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}
QLabel
{
color: rgb(0, 255, 255);
background-color: rgb(0, 0, 0);
border-style: outset;
border-width: 3px;
border-radius: 15px;
border-color: rgb(255, 0, 0);
padding: 2px;
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;여기서 보여져야...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
